content/docs/components/serving/seldon.md

```diff
@@ -49,12 +47,9 @@ spec:
 ```
 
 Save the above resource and apply it with `kubectl`.
- 
 
-### Examples
+## Examples
 
-   * [Kubeflow Seldon E2E Pipeline](https://docs.seldon.io/projects/seldon-core/en/latest/examples/kubeflow_seldon_e2e_pipeline.html)
+* [Kubeflow Seldon E2E Pipeline](https://docs.seldon.io/projects/seldon-core/en/latest/examples/kubeflow_seldon_e2e_pipeline.html)
 
 Seldon provides a [large set of example notebooks](https://docs.seldon.io/projects/seldon-core/en/latest/examples/notebooks.html) showing how to run inference code for a wide range of machine learning toolkits.
```

Comment:
Looking at the "Examples" section, I'm now thinking it'd  be good to make the last item a bullet point too, to match the one on line 53. If we do that, then we need to change the item on line 53 to a full sentence. Suggestion for the item on line 53:

```
* The  [Kubeflow Seldon E2E Pipeline](https://docs.seldon.io/projects/seldon-core/en/latest/examples/kubeflow_seldon_e2e_pipeline.html) shows how to build re-usable components for an ML pipeline.
```

What do you think?

Reply:
I completely agree. The one bullet point seemed somewhat odd.

Reply:
What do you think about no list for the `Examples`? Easy enough to put back.

Reply:
That looks good!